pkg/batcher/handler.go

```diff
@@ -29,10 +30,10 @@ import (
 	"time"
 )
 
-const (
-	SleepTime    = time.Microsecond * 100
-	MaxBatchSize = 32
-	MaxLatency   = 5000
+var (
+	SleepTime, _    = utils.LookupEnvDurationOr("BATCH_HANDLER_SLEEP_TIME", time.Microsecond*100)
+	MaxBatchSize, _ = utils.LookupEnvIntOr("BATCH_HANDLER_MAX_BATCH_SIZE", 32)
+	MaxLatency, _   = utils.LookupEnvIntOr("BATCH_HANDLER_MAX_LATENCY", 5000)
```

Comment:
These are default values and can be overwritten by the `MaxBatchSize` and `MaxLatency` on the inference service yaml. They shouldn't be environment variables, we can make t these defaults configurable in the inference service ConfigMap [here](https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L288).

Reply:
Thanks. I missed that and was following a practice from another project. I'll revise the PR to make these configurable via configmap.

Reply:
Updated the PR. Please take another look. Thanks.